src/features/home/redux/connectWallet.js

```diff
@@ -42,7 +42,8 @@ export function connectWallet(web3Modal) {
             dispatch(disconnectWallet(web3, web3Modal));
           }
         });
-        provider.on('networkChanged', async networkId => {
+        provider.on('chainChanged', async chainId => {
```

Comment:
`networkChanged` not working with Binance and deprecated in Metamask in favor of `chainChanged`, which returns hex number